src/shared/models/asset/asset.service.ts

```diff
@@ -8,6 +8,7 @@ export class AssetService {
   constructor(private assetRepo: AssetRepository) {}
 
   async getAllAsset(blockchain: Blockchain): Promise<Asset[]> {
+    if (!blockchain) blockchain = Blockchain.DEFICHAIN;
```

Comment:
What do you think about a default parameter? Did you explicitly not use it?
```ts
  async getAllAsset(blockchain: Blockchain = Blockchain.DEFICHAIN): Promise<Asset[]> {
    return this.assetRepo.find({ where: { blockchain } });
  }
```

Reply:
`getAllAsset` is being called with `jwt.blockchain` means, if somebody has no blockchain in there jwt yet, this function is being called with undefined, and thats' why I am checking it this way.

Tbh I am not sure, how default values behaves, if someone explicitly is calling it with `undefined`